### PR TITLE
Fix/Optional Settings Override

### PR DESCRIPTION
**Pull request by samchristensen**
**Description**
The SDK does not provide an implementation for the `BinLookup` feature. However, when attempting to create an implementation for myself, the way that versions are managed in the `client.py` does not allow for me to easily override the API version being used for the `BinLookup` service.

This pull request allows for optionally providing settings to the `AdyenClient` class, which in turn will allow developers to better control which API version they are using if need be.

### CODE REVIEW

**Comment by samchristensen** [comment]
> Does adding the `API_BIN_LOOKUP_VERSION` to the `settings.py` solves your issue? I'm not totally in favor of allowing the settings to be overwritten.

Yeah, we could definitely go the route of adding `API_BIN_LOOKUP_VERSION` to the settings.py, and then adding an additional check such as this in the `client.py`.

```
elif service == "Bin"
    api_version = settings.API_BIN_LOOKUP_VERSION
``` 

If maintaining control on the settings used for this library, I think this is the most obvious solution.

File: Adyen/settings.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
 BASE_PAL_URL = "https://pal-{}.adyen.com/pal/servlet"
 BASE_HPP_URL = "https://{}.adyen.com/hpp"
 ENDPOINT_CHECKOUT_TEST = "https://checkout-test.adyen.com"
-ENDPOINT_CHECKOUT_LIVE_SUFFIX = "https://{}-checkout-live" \
-                                ".adyenpayments.com/checkout"
+ENDPOINT_CHECKOUT_LIVE_SUFFIX = "https://{}-checkout-live.adyenpayments.com/checkout"
```
Comment (caioariede) [inline Adyen/settings.py]: @samchristensen can we undo this unrelated change?

**Comment by rikterbeek** [comment]
HI @samchristensen,

With the library we want to control the API version that is used because we then know if it works with the logic that is build into the library. We want to update the libraries always to the latest API version. If you want to update the API version to the latest version that we totally understand. Just provide a PR with the latest API version and we will validate it agains all our test and do a new release so all our merchants can benefit from this.

Regards,
Rik
Adyen
